exercises/luhn-trait/example.rs

```diff
@@ -0,0 +1,55 @@
+pub trait Luhn {
+    fn valid_luhn(&self) -> bool;
+}
+
+impl Luhn for String {
+    fn valid_luhn(&self) -> bool {
+        if self.chars().any(|c| c.is_alphabetic()) || self.chars().count() == 1 {
+            return false;
+        }
+
+        self.chars()
+            .filter_map(|c| c.to_digit(10))
+            .rev()
+            .enumerate()
+            .map(|(index, digit)| if index % 2 == 0 { digit } else { digit * 2 })
+            .map(|digit| if digit > 9 { digit - 9 } else { digit })
+            .sum::<u32>() % 10 == 0
+    }
+}
+
+impl Luhn for &'static str {
```

Comment:
same - I'd ask if there is any requirement that these be `&'static str` rather than just `&str`.